.github/workflows/release-check.yml

```diff
@@ -10,9 +10,60 @@ jobs:
   release-file-check:
     runs-on: ubuntu-latest
 
+    outputs:
+      changelog: ${{ steps.release-check.outputs.changelog }}
+      status: ${{ steps.release-check.outputs.release_status }}
+      change_type: ${{ steps.release-check.outputs.change_type }}
+
     steps:
     - uses: actions/checkout@v1
 
     - name: Release file check
       uses: ./.github/release-check-action
+      id: release-check
       if: github.event.pull_request.draft == false
+
+  generate-preview:
+    runs-on: ubuntu-latest
+    needs: release-file-check
+    if: always() && github.event.pull_request.draft == false
+
+    outputs:
+      url: ${{ steps.upload.outputs.url }}
+
+    steps:
+    - name: Generate Twitter card preview
+      uses: strawberry-graphql/release-cards@main
+      if: ${{ needs.release-file-check.outputs.status == 'OK' }}
+      with:
+        version: "(next)"
+        contributor: ${{ github.event.pull_request.user.login }}
+        description: ${{ fromJSON(needs.release-file-check.outputs.changelog) }}
+
+    - name: Uploads preview
+      id: upload
+      if: ${{ needs.release-file-check.outputs.status == 'OK' }}
+      run: |
+        echo ::set-output name=url::$(curl --location --request POST "https://api.imgur.com/3/image" \
+          --fail \
+          --header "Authorization: Client-ID $IMGUR_CLIENT_ID" \
+          --form "image=@screenshot.png;filename=screenshot.png;type=image/png" \
+          | jq  --raw-output '.data.link')
+      env:
+        IMGUR_CLIENT_ID: ${{ secrets.IMGUR_CLIENT_ID }}
+
+  send-comment:
+    runs-on: ubuntu-latest
+    needs: [release-file-check, generate-preview]
+    if: always() && github.event.pull_request.draft == false
```

Comment:
Why the always()?

Reply:
I always want to send the comment, even if any of the previous job fail (the release file check fails if the file is missing or wrong) :)